src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,22 @@
+package racingcar.controller;
+
+import racingcar.service.CarRacing;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
+
+public class RacingCarController {
+    private static final InputView inputView = new InputView();
+
+    public static void main(String[] args) {
+        int numberOfCars = inputView.enterNumberOfCars();
+        int numberOfMoves = inputView.enterNumberOfMoves();
+
+        CarRacing carRacing = new CarRacing(numberOfCars);
+        ResultView resultView = new ResultView(carRacing);
+
+        for(int i = 0; i < numberOfMoves; i++) {
+            carRacing.carRaceStart();
+            resultView.printCarsStatusResult();
+        }
```

Comment:
```suggestion
        CarRacing carRacing = new CarRacing(numberOfCars);
        ResultView resultView = new ResultView();

        for(int i = 0; i < numberOfMoves; i++) {
            List<CarRecord> records = carRacing.carRaceStart();
            resultView.printCars(records);
        }
```

이런 방향으로 변경해도 좋을 것 같지 않나요?